src/libs/vlayout/vlayoutpiece_p.h

```diff
@@ -116,8 +116,8 @@ class VLayoutPieceData : public QSharedData
     /** @brief layoutAllowance list of layout allowance points. */
     QVector<QPointF>          layoutAllowance;
 
-    /** @brief passmarks list of passmakrs. */
-    QVector<QLineF>           passmarks;
+    /** @brief notches list of passmakrs. */
```

Comment:
passmakrs?

Reply:
Ah... this can be notches. If you notice  passmakrs is misspelled. Hard to anticipate misspellings.